src/components/Footer/FooterBase.tsx

```diff
@@ -0,0 +1,128 @@
+import Link from '../Link';
+import logoTagline from '../../stories/assets/logo-bsc-tagline.svg';
+import { RequireOnlyOne, NavItem } from '../../types';
+import Facebook from '../../stories/assets/icon-facebook-logo.svg';
+import Twitter from '../../stories/assets/icon-twitter-logo.svg';
+import LinkedIn from '../../stories/assets/icon-linkedin-logo.svg';
+import YouTube from '../../stories/assets/icon-youtube-logo.svg';
+
+interface LegalLinksBase extends NavItem {
+  children?: LegalLinkItem[];
+}
+
+export type LegalLinkItem = RequireOnlyOne<LegalLinksBase, 'to' | 'href'>;
+
+export interface FooterBaseProps {
+  legalLinkItems?: LegalLinkItem[];
+  legalLinkAriaLabel?: string;
+  corporateLink?: boolean;
```

Comment:
do we need an aria label here too?

Reply:
I believe so, there are two nav elements within the region. lmk if I'm wrong 😅